boxy/lib/src/boxy_flex.dart

```diff
@@ -438,7 +438,7 @@ class BoxyFlexible extends ParentDataWidget<FlexParentData> {
     }
 
     if (needsLayout) {
-      final AbstractNode? targetParent = renderObject.parent;
```

Comment:
renderObject.parent changed from `AbstractNode?` to `RenderObject?` in flutter 3.13.0. However, explicitly specifying the variable type will break older flutter versions. 

Just leaving it resolve implicitly works for both flutter <3.13.0 and >=3.13.0.